src/onboarding-component/tests/index.test.jsx

```diff
@@ -326,4 +329,31 @@ describe('OnBoardingComponent Test', () => {
     const { getByTestId } = render(reduxWrapper(<IntlOnBoardingComponent isOpen close={() => {}} />));
     expect(getByTestId('tpa-spinner')).toBeTruthy();
   });
+
+  it('should have called with forceRefresh true', async () => {
+    const user = {
+      username: 'gonzo',
+      other: 'data',
+    };
+
+    getAuthenticatedUser.mockReturnValue(user);
+    fetchAuthenticatedUser.mockReturnValueOnce(Promise.resolve(user));
+
+    await act(async () => {
+      await render(reduxWrapper(<IntlOnBoardingComponent isOpen close={() => {}} />));
```

Comment:
You can remove this `await`.

Reply:
i tried removing this await `await` but tests were not running successfully that is why i kept it as it is